source/authentication/email-password.txt

```diff
@@ -281,29 +277,29 @@ Send a Password Reset Email
 
 If you configure the provider to :guilabel:`Send a Password Reset Email`,
 {+service-short+} sends a Password Reset URL to a user's email address when a
-user requests a password reset. The user must visit this URL within 30 minutes
+{+service-short+} sends a Password Reset URL to the user's email address on a password reset request. The user must visit this URL within 30 minutes
 to confirm their intent to change their password.
 
-Configure the following settings to have {+service-short+} automatically send a
-password reset email:
+When you configure password reset emails, you can specify these settings:
+configure the following settings:
 
 - :guilabel:`Password Reset URL`: the base of the URL included in every
   password reset email. {+service-short+} appends a unique ``token`` and
-  ``tokenId`` to this URL as query parameters to create a unique link
+  ``tokenId`` to this URL. These serve as query parameters to create a unique link
   for every password reset. To reset the user's password, extract these
-  query parameters from the user's unique URL and pass the token and
+  query parameters from the user's unique URL. Pass the token and
   tokenId to the Client SDK's ``resetPassword`` function.
 
 - :guilabel:`Reset Password Email Subject`: the subject line of the
-  email that {+service-short+} sends to users when they request to reset their
-  password. This value is optional: if omitted, {+service-short+} uses a default
+  - :guilabel:`Reset Password Email Subject`: the subject line for the email that App Services sends. 
```

Comment:
Looks like we've got some more duplicate content here. If we're going with line 294, we should delete line 293. (And we should probably also wrap ln 294 per the line length guidelines in our style guide.)